incubator/command-dump/src/test/java/io/aklivity/zilla/runtime/command/dump/internal/GrpcServerStreamRpcIT.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.command.dump.internal;
+
+import static io.aklivity.zilla.runtime.engine.EngineConfiguration.ENGINE_TIMESTAMPS;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.junit.rules.RuleChain.outerRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.DisableOnDebug;
+import org.junit.rules.TestRule;
+import org.junit.rules.Timeout;
+import org.kaazing.k3po.junit.annotation.Specification;
+import org.kaazing.k3po.junit.rules.K3poRule;
+
+import io.aklivity.zilla.runtime.command.dump.internal.test.DumpRule;
+import io.aklivity.zilla.runtime.engine.test.EngineRule;
+import io.aklivity.zilla.runtime.engine.test.annotation.Configuration;
+
+public class GrpcServerStreamRpcIT
+{
+    private final K3poRule k3po = new K3poRule()
+        .addScriptRoot("net", "io/aklivity/zilla/specs/command/dump/binding/grpc/streams/network/server.stream.rpc")
+        .addScriptRoot("app", "io/aklivity/zilla/specs/command/dump/binding/grpc/streams/application/server.stream.rpc");
+
+    private final TestRule timeout = new DisableOnDebug(new Timeout(10, SECONDS));
+
+    private final EngineRule engine = new EngineRule()
+        .directory("target/zilla-itests")
+        .countersBufferCapacity(4096)
+        .configure(ENGINE_TIMESTAMPS, false)
+        .configurationRoot("io/aklivity/zilla/specs/command/dump/binding/grpc/config")
+        .external("app0")
+        .clean();
```

Comment:
Apart from `tcp` and `filesystem` binding tests, which are terminal in the bindings pipeline, do we actually need to have the engine be present in any of the other ITs?

It seems like we might be re-testing the various binding implementations instead of just the dump command output, agree? Is there any benefit to that?

Reply:
Actually, if we are using both sides of the app scripts through k3po to generate the ringbuffer that will be picked up by the dump command for the ITs, then we don't need the EngineRule for any of the ITs, right?

This should also make the ordering less relevant because all the frames will be in `data63` created by k3po.

Reply:
You are making a very interesting point. I think what you are saying is k3po alone should be able to do a green test run "by talking to itself" without the engine being present, right? I tried removing the engine rule, but k3po tests didn't complete successfully in that case, not sure why.

Reply:
In any case, a strong argument for runnning the Engine is the `bindings` file which is created by the Engine at startup. The Engine reads the config, extracts the necessary metadata and writes it to the `bindings` file. That's a hard requirement for the dump command to work as it doesn't have any knowledge about the config file.